src/com/rakovets/course/challenge/exception/Task03.java

```diff
@@ -0,0 +1,79 @@
+package com.rakovets.course.challenge.exception;
+
+public class Task03 {
+    public static void main(String[] args) {
+        Car car = new Car("Bugatti");
+        System.out.println(car.getName());
+        car.setHasWheels(true);
+        car.setHasSteeringWheel(true);
+        car.setHasChassis(true);
+        car.setHasWaterPillow(false);
+        car.setHasСaterpillars(false);
+        try {
+            car.Drive();
+        } catch (ExceptionDravingCar exceptionDravingCar) {
+            System.out.println(exceptionDravingCar.getMessage());
+            System.out.println("Checkmanagement mechanisms of car !, Car have a Chassis " + "\n" + car.hasChassis + "\n" + "car has Wheels " +
+                    "\n" + car.hasWheels + "\n" + "Car has SteeringWheel\n" + "\n" + car.hasSteeringWheel + "\n" + "Car has Сaterpillars" + "\n" +
+                    car.hasСaterpillars + "\n" + "Car has WaterPillow " + "\n" + car.hasWaterPillow);
+            exceptionDravingCar.printStackTrace();
+        }
+    }
+
+    public class ExceptionDravingCar extends RuntimeException {
```

Comment:
Best practice for naming:
```suggestion
    public class DrivingCarException extends RuntimeException {
```